pkg/expr/query.go

```diff
@@ -96,6 +96,9 @@ type ReduceSettings struct {
 type ReduceMode string
 
 const (
+	// Default mode
+	ReduceModeStrict = ""
```

Comment:
hey sorry I'm out of the loop what is strict mode?

Reply:
hi, it's a setting for a `reduce` expression:
<img width="703" alt="reduce-mode" src="https://github.com/user-attachments/assets/370441ac-c37b-4c7d-8cd3-c24c31bc7fb2">

`strict` mode is the default, and the corresponding json-string-value is `""`

(there is a bit of complication that we don't need to handle here... this `""` value only happens if you open that selectbox, choose something else, then choose the default-value again... if you NEVER open the select-box, this mode-thing will not even be in the json-structure.. so, for strict-mode there are two possible representations 😢 )